e2e/basic/file_test_image_cached_test.go

```diff
@@ -37,6 +37,10 @@ func TestFileCached(t *testing.T) {
 		if err != nil {
 			t.Fatalf("Error executing command for '%v': %v", instanceName, err)
 		}
+		err = instance.AddVolumeWithOwner("/usr/share/nginx/html", "1Gi", 0)
```

Comment:
Anytime we fix something, we should also add a comment. 
Better yet, this entire section of code in this for loop seems like a common test pattern and so should be refactored into a helper. 
Then the helper should be nicely documented explaining why the helper does and why it needs to do it so that future tests can understand how and why to use it.